store/sqlstore/user_store.go

```diff
@@ -492,67 +488,54 @@ func (us SqlUserStore) InvalidateProfilesInChannelCache(channelId string) {
 
 func (us SqlUserStore) GetProfilesInChannel(channelId string, offset int, limit int) store.StoreChannel {
 	return store.Do(func(result *store.StoreResult) {
-		var users []*model.User
+		query := usersQuery.
+			Join("ChannelMembers cm ON ( cm.UserId = u.Id )").
```

Comment:
Switching to a `JOIN` here from an additional `FROM` table. No semantic differences.